src/Employer/Employer.Web/Orchestrators/VacanciesOrchestrator.cs

```diff
@@ -154,13 +152,17 @@ private FilteringOptions SanitizeFilter(string filter)
 
         private string GetFilterHeading(int totalVacancies, FilteringOptions filteringOption, string searchTerm)
         {
+            string vacancyStatusPrefix;
             var filterText = filteringOption == FilteringOptions.All ? string.Empty : $" {filteringOption.GetDisplayName().ToLowerInvariant()}";
-            var vacancyText = filteringOption == FilteringOptions.ClosingSoon || filteringOption == FilteringOptions.ClosingSoonWithNoApplications ?
+            var vacancyText = filteringOption == FilteringOptions.Live || filteringOption == FilteringOptions.ClosingSoon || filteringOption == FilteringOptions.ClosingSoonWithNoApplications ?
                 " live vacancy" : " vacancy";
-            var vacancyStatusPrefix = $"{totalVacancies}{vacancyText}".ToQuantity(totalVacancies, ShowQuantityAs.None) + $"{filterText}";
-
+            if(filteringOption == FilteringOptions.Live)
+                vacancyStatusPrefix = $"{totalVacancies}{vacancyText}".ToQuantity(totalVacancies, ShowQuantityAs.None);
```

Comment:
This could be simplified as
`vacancyStatusPrefix = vacancyText.ToQuantity(totalVacancies);`
No need for leading space in the vacancyText.